Assets/Mirror/Runtime/NetworkWriter.cs

```diff
@@ -154,6 +166,16 @@ public void WriteUInt64(ulong value)
         }
 
         public void WriteInt64(long value) => WriteUInt64((ulong)value);
+
+        /// <summary>
+        /// Writes any type that mirror supports
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="msg"></param>
+        public void Write<T>(T msg)
```

Comment:
```suggestion
        public void Write<T>(T value)
```

most functions in this class use `value` for the arg name